### PR TITLE
use magic constant and core function differently

### PR DESCRIPTION
**Pull request by nathansmonk**
This request will allow the plugin to work regardless of folder name

### CODE REVIEW

**Comment by nathansmonk** [comment]
Do I need to do anything else to get this accepted?

**Comment by ChristopherDBerry** [comment]
Looks OK

File: Sitemorse-SCI/readme.txt [hunk 1]
```diff
@@ -35,16 +35,15 @@ before it is published and will give you a chance to fix any issues.
...
 3. Activate the plugin through the 'Plugins' screen in WordPress.
 4. Enter the licence key in the Settings->Sitemorse section. Normally other settings can be left with the default values.
 5. For further information please contact your Sitemorse representative.
 
 
 == Testing ===
 Running an assessment on any piece of content will test the full connection. For
-example, from the assessment button whilst editing content or the Sitemorse
-button on the admin bar.
+example,
```
Comment (jribbens) [inline Sitemorse-SCI/readme.txt]: There seems to be something missing here.

**Comment by jribbens** [comment]
I've put a comment on the readme file diff, if this is resolved I will then merge this pull request.

File: Sitemorse-SCI/readme.txt (hunk 1 above)
Comment (nathansmonk) [reply Sitemorse-SCI/readme.txt]: whoops! Sorry, not sure whats happened there. Adding the original back in

**Comment by jribbens** [comment]
Merged; thank you @nathansmonk

**Comment by nathansmonk** [comment]
Woo hoo 🎉 Thanks!

**Pull request merged**